### DISCUSSION TITLE
Weird flash on input focus

### DISCUSSION THREAD

**Discussion by captenmasin** [tailwindlabs/tailwindcss#3685] [Help] [answered]
I'm not sure if this is actually an issue but I've noticed with my inputs (text and buttons) I get this weird flash on focus?

https://user-images.githubusercontent.com/25454872/110131920-90b76e80-7dc2-11eb-9324-5c7c4d82d5e9.mp4

Classes:
```css
    flex items-center px-4 py-2 mx-auto text-sm font-medium transition-all rounded-lg w-full justify-center
    group pl-12 pr-4
    focus:outline-none
    focus:ring-0
```

Has anyone else experienced this?

**Comment by simonswiss** [comment, marked as answer]
Hey!

This is because you're using the `transition-all` [transition property](https://tailwindcss.com/docs/transition-property) class, which will transition **every** CSS property over the default duration (150ms).

So, basically your `outline-none` transitions over 150ms as well, hence the weird flash you see when entering/leaving focus.

You probably want to change this to something like just `transition`, or one of the other values.

Hope that makes sense 👍

**Reply by captenmasin** [reply]
Ooooh that was exactly it! Thank you so much, that's fixed it <3
